### ISSUE TITLE
`match` does not go through transactions sequentially

### ISSUE THREAD

**Issue by ohbadiah** [johannesgerer/buchhaltung#7]
`buchhaltung match` seems to only go through the odd numbered transactions in its queue. Here's a snippet:

```
Current Transaction: 27, Remaining: 57
uncertainty: 4.88	Groceries:Food:Expenses
[<, >, save, RET]:
Learning: Expenses:Food:Groceries
Done:     Expenses:Food:Groceries
Field            | Value
---------------- | --------------------
Amount           | -2.50
Category         | DEBIT
Description      | PPA ON STREET METERS
Transaction Date | 04/09/2017
formatName       | barclaycard
formatVersion    | May 2017

Current Transaction: 29, Remaining: 55
uncertainty: 4.96	Parking:Car:Transportation:Expenses
[<, >, save, RET]:
Learning: Expenses:Transportation:Car:Parking
Done:     Expenses:Transportation:Car:Parking
Field         | Value
------------- | ----------------------------------
Date          | 04/11/2017
Description   | ACH WEBRECUR XXXXX4679PA AES STDNT
              | LOAN
Withdrawals   | $123.45
formatName    | pncbank
formatVersion | May 2017

Current Transaction: 31, Remaining: 53
```

Navigation using `<` and `>` is still possible.

**Comment by johannesgerer** [comment]
fixed in nick branch. will be merged soon

**Comment by varac** [comment]
@johannesgerer is this fix included in any release ?

**Comment by johannesgerer** [comment]
yes. The code change that (I think) fixes this is in 0.0.7.
